source/integrate_with_govuk_pay/index.html.md.erb

```diff
@@ -76,6 +76,20 @@ You could also connect a customer-relationship management (CRM) or case manageme
 
 Read more about [reporting](/reporting/#reporting) and [refunding payments](/refunding_payments/#refunding-payments).
 
+### When you'll receive payments
+
+You'll receive the money in your bank account within 2 business days of your user completing their payment.
+
+It will take 3 business days if your user completed their payment:
+
+- after midnight UTC on a weekday
```

Comment:
this would mean it's on the next working day and so is on the regular payout schedule. ie if it's after midnight on monday, it's really tuesday, so payout is on thursday. So we can cut this line I think and just leave the weekend and bank holiday info?

Reply:
I think this line was was there for users who need to know that payments made at midnight BST (11pm UTC) will take 2 days rather than 3?

Reply:
As discussed - removed in https://github.com/alphagov/pay-tech-docs/pull/374/commits/b5ed8c144e88f9b620bb9b28f0664a6be40cc253.